src/iter/par_bridge.rs

```diff
@@ -126,8 +119,7 @@ where
 
         loop {
             // Check if the iterator is exhausted *and* we've consumed every item from it.
```

Comment:
The `&&` part is gone now that each split is pulling one item at a time for their own consumption.

```suggestion
            // Check if the iterator is exhausted.
```